core/feed/annotator/base.py

```diff
@@ -304,6 +304,9 @@ def annotate_work_entry(
         if edition.series:
             computed.series = self.series(edition.series, edition.series_position)
 
+        if edition.duration is not None:
```

Comment:
Codecov is saying these lines aren't covered. Is it just being broken, or are the new tests bypassing this line somehow?

Reply:
Didn't create a test of a serialisation without duration, will create one for completeness